cleanlab/classification.py

```diff
@@ -313,16 +321,17 @@ class 0, 1, ..., K-1. `pred_probs` should have been computed using 3 (or
           that are mislabeled examples from every other class ``k_y``,
           Assumes columns of `inverse_noise_matrix` sum to 1.
 
-        label_issues_mask : np.array, optional
-          A boolean mask for the entire dataset such as those returned by:
+        label_issues : pd.DataFrame or np.array, optional
+          Specifies the label issues for each example in dataset.
+          If DataFrame, must be formatted as the one returned by:
           :py:meth:`CleanLearning.find_label_issues
-          <cleanlab.classification.CleanLearning.find_label_issues>` or
+          <cleanlab.classification.CleanLearning.find_label_issues>` or by this function.
+          If np.array, must contain boolean label_issues_mask as output by:
```

Comment:
While we're at it, why not also support an array of integers, in case the user got ranked label issues back from `find_label_issues`?

Reply:
added support for that, with check to make sure they're all unique